### DISCUSSION TITLE
桌面歌词 有没有可能适配热词

### DISCUSSION THREAD

**Discussion by SouL87a** [SPlayer-Dev/SPlayer#128] [想法 Ideas]
在已知的所以 音乐软件 官方或者第三方的桌面歌词，感觉 [热词](https://[github.com/cnbluefire/HotLyric](https://github.com/cnbluefire/HotLyric))是最好的。。。。并且热词很容易接入，只要支持smtc，然后热词那边接入一些就行了。
感觉没必要自己重新写 桌面歌词，重复造轮子了。

**Comment by SouL87a** [comment]
看来眼花了，刚才看到后面会支持桌面歌词，后面又看到 作者说 不会添加的功能的中有桌面歌词。。

打扰了

**Reply by imsyy** [reply]
没事，是说不会写这个功能，适配倒是可以

**Comment by serious-snow** [comment]
热词支持mac吗
